py/server/deephaven/perfmon.py

```diff
@@ -95,6 +95,38 @@ def query_performance_log() -> Table:
     except Exception as e:
         raise DHError(e, "failed to obtain the query performance log table.") from e
 
+def query_operation_performance_log_as_tree_table() -> TreeTable:
```

Comment:
```suggestion
def query_operation_performance_tree_table() -> TreeTable:
```
Name is already way way long.